src/composed/message/decrypt.rs

```diff
@@ -14,7 +14,7 @@ where
     debug!("decrypt session key");
 
     locked_key.unlock(key_pw, |priv_key| {
-        let (key, sym_alg) = priv_key.decrypt(mpis, &locked_key.fingerprint())?;
+        let (key, sym_alg) = priv_key.decrypt(mpis, locked_key.fingerprint().as_bytes())?;
         // TODO: handle other versions
```

Comment:
uhm, what about this todo?

Reply:
The missing functionality for this todo will be part of the followup PR that will handle v6 encryption